src/Dapr.Client/DaprClient.cs

```diff
@@ -233,6 +233,12 @@ public HttpRequestMessage CreateInvokeMethodRequest(string appId, string methodN
         /// <returns>An <see cref="HttpRequestMessage" /> for use with <c>SendInvokeMethodRequestAsync</c>.</returns>
         public abstract HttpRequestMessage CreateInvokeMethodRequest(HttpMethod httpMethod, string appId, string methodName);
 
+        /// <summary>
+        /// Creates an <see cref="HttpRequestMessage" /> that can be used to perform health-check of Dapr sidecar
+        /// </summary>
+        /// <returns>An <see cref="HttpRequestMessage" /> for use with <c>InvokeHealthMethodAsync</c>.</returns>
+        public abstract HttpRequestMessage CreateInvokeHealthMethodRequest();
```

Comment:
What's the use case of this separate from `InvokeHealthMethodAsync`? If there isn't a concrete scenario where this is necessary, just leave it out - it could be added latter if needed.

Reply:
currently no scenario, just try to be consistent with request methods. Will simplify